firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -856,10 +855,14 @@ class SearchViewController: SiteTableViewController,
         case .openedTabs:
             return filteredOpenedTabs.count
         case .remoteTabs:
-            return model.shouldShowSyncedTabsSuggestions ?
-            filteredRemoteClientTabs.count : 0
+            return if featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser) {
+                model.shouldShowSyncedTabsSuggestions ?
+                filteredRemoteClientTabs.count : 0
+            } else { filteredRemoteClientTabs.count }
         case .bookmarksAndHistory:
-            return numberOfItemsFromData
+            return featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser) ?
+                   numberOfItemsFromData :
+                   data.count
```

Comment:
hahaha this is a third way you're splitting ternary operator in this PR. Can this be improved?

Reply:
I split it based on length of the line.